spec/generator_spec.rb

```diff
@@ -14,32 +14,32 @@
   end
 
   context "without flags" do
-    let(:command) { "bundle exec rake install_all && bin/decidim #{test_app}" }
+    let(:command) { "bundle exec decidim #{test_app}" }
```

Comment:
This is actually necessary to help preventing regressions in generators. The idea is to simulate real usage where they will be no bundler context and the user might only have decidim dependencies installed on her system (via `gem install decidim`). I'm going to open a PR to help clarifying these specs a bit.

Reply:
Oh okay! Let me check how this works. By the way, I'm working in parallel on https://github.com/decidim/docker to be able to rationalize usage of docker images!

Reply:
Yeah, I'm getting notifications :) Outstanding work :+1: :+1:

Reply:
I added #2217 to try to shed some light on these weird specs...